backend/prompt_studio/prompt_studio_v2/models.py

```diff
@@ -84,6 +84,17 @@ class Mode(models.TextChoices):
         db_comment="Field to store the prompt key",
         unique=False,
     )
+    REQUIRED_CHOICES = [
```

Comment:
[MINOR] For readability, prefer the CHOICES being declared under the class name.

Reply:
done. @harini-venkataraman